rllib/core/learner/learner.py

```diff
@@ -111,12 +112,51 @@ class LearnerHyperparameters:
 
     Validation of the values of these hyperparameters should be done by the
     respective AlgorithmConfig class.
+
+    For configuring different learning behaviors for different (single-agent) RLModules
+    within the Learner, RLlib uses the `per_module_overrides` property (dict), mapping
+    ModuleID to a "sub-version" of a `LearnerHyperparameters` instance, in which the
+    module-specific override settings are applied AND the `per_module_overrides` is set
+    to None.
     """
 
-    # TODO (Sven): Move lr from - currently - optimizer config to only exist here.
-    # lr: float = None
+    # Parameters used for gradient postprocessing (clipping) and gradient application.
+    optimizer_type: str = None
```

Comment:
4 base settings supported by RLlib for auto-constructing one optimizer per module.

If users want any other behavior:
- more than 1 optimizer per module
- custom type of optimizer with custom settings (e.g. Adam epsilon, etc..)
they must override `configure_optimizers_for_module` and then RLlib will NO longer manage the lr/lr-schedule for them. Everything has to be done manually then by the user's own Learner subclass.